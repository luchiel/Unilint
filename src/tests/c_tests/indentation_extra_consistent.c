int main() {
        //this is extra-indented block
    //and this one is badly aligned
    }