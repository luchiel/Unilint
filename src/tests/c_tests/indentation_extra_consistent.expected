3:1: indent too small
